v0.3/tutorials/quickstart.eve

```diff
@@ -7,117 +7,95 @@ title: Quickstart
+By making a button for each page from the list, it's easy to add more pages later. For now though, we're going to focus on the Home page. The first step is to make a special record to keep track of which page we're looking at.
 
-~~~eve
-search
-  event = [#html/event/click element: [#increment]]
+## Starting Page
 
+```
 commit
-  [#clicked event]
-~~~
-
-Clicks only last for an instant, but we want to create a permanent record of each click so we can search for them later. This block commits a `#clicked` record that will persist until it's explicitly removed. Much like the `#greeting` text we bound to the `#ui`, variables with the same name are equivalent, so the variable `event` in the `#clicked` record is a reference to the `#html/event/click` on the `#increment` button.
-
-The identity of a record is determined by its attribute/value pairs. Two records with the same attributes and values are the same record in Eve. We included the `event` attribute in the `#clicked` record to differentiate each record. Without this differentiation, we could only ever create a single `#clicked` record.
+  [#app page: "Home"]
+```
 
-## Count the number of clicks
+This makes Home the default starting location. Now that we're looking at Home, we can add some content to it.
 
-Now let's count the number of times the button has been clicked. Make sure `event` is back in `#clicked`, and then we can count those records directly:
+## Home Page
 
-~~~eve
+```
 search
-  how-many = gather/count[for: [#clicked]]
+  [#app page: "Home"]
+  view = [#contents]
 
 bind
-  [#ui/text text: "The button has been clicked {{how-many}} times"]
-~~~
-
-This block searches for every unique `#clicked`, counts them, and returns that value in `how-many`. Then we display this value in a text container using the operator `{{ ... }}`, which inserts the value of the contained variable into the string. An important thing to remember here is that this block will only run when the button has been clicked at least once. Before then, this block will not run because there are no `#clicked` records to count.
-
-## Summary
-
-That's it for the 5 minute introduction to Eve. To summarize:
+  view.children += ([#ui/div text: "This is the Eve Quickstart app! Choose a link to see some of the things Eve can do."], [#ui/img src:"http://i.imgur.com/qup3Y7l.jpg"])
+```
 
-- Eve programs are made up of blocks.
-- Data are represented by records, key value pairs associated to a unique ID.
-- There are two sections of a block: one where you search for records, and one where you bind or commit records.
-- Blocks update records automatically to reflect changes in data.
-- Bound records are replaced when data changes, while committed records must be removed manually.
-- Records are unique, uniqueness is determined by a record's attributes and their values.
+## Navigation
+The last thing to do is make the nav bar links switch which page the app is showing.
```

Comment:
strongly disagree. trying to teach eve concepts in the qs is what got us into trouble to begin with - here, they expect to click a button and see the page change, which is what happens, so we shouldn't clutter it up with explanations for things that already make sense. click event clean-ups get taught very shortly down the road with the counter example, which fits in more organically